package.json

```diff
@@ -24,6 +24,9 @@
     "clean": "rimraf build",
     "clean:modules": "rimraf node_modules"
   },
+  "engines": {
+    "node": ">=12.0.0"
```

Comment:
This action actually runs on Node 16 (you can see this in the `action.yml` file) so that's what we should use here